desktop/src/renderer/hooks/useNotification/useNotification.tsx

```diff
@@ -0,0 +1,92 @@
+import { Toast } from "@renderer/components";
+import { toast, ToastContainer, type ToastContainerProps } from "react-toastify";
+// import "react-toastify/dist/ReactToastify.css";
+import { getXlsxFileListNotificationData } from "./getXlsxFileListNotificationData";
+
+/**
+ * Props for notification messages.
+ */
+export interface NotifyProps {
+	success: boolean;
```

Comment:
Instead of passing in success each time, could you just pass success: true when they use notify.success and pass success: false when they use notify.error ?

Reply:
I have implemented this